### ISSUE TITLE
Clarify the alignment of pOffsets passed to vkCmdBindVertexBuffers 

### ISSUE THREAD

**Issue by godlikepanos** [KhronosGroup/Vulkan-Docs#2411]
The documentation of vkCmdBindVertexBuffers doesn't mention something about the alignment of pOffsets. Not sure if the alignment requirements are mentioned in another section but I think it would be a good idea to have something as part of vkCmdBindVertexBuffers's documentation.

**Comment by HansKristian-Work** [comment]
Alignment is validated after all vertex addressing logic has been applies, so it's too early to attempt validation at VBO bind time.

**Comment by godlikepanos** [comment]
Right, true. I wonder if we can add some text that points to https://registry.khronos.org/vulkan/specs/1.1-extensions/html/vkspec.html#fxvertex-input-extraction for additional clarity.

**Comment by oddhack** [comment]
@godlikepanos could you propose a PR adding a NOTE to that section, based on @HansKristian-Work's comment?

**Comment by godlikepanos** [comment]
Sorry for the late update. I'll look into it.
